### ISSUE TITLE
[Accessibility] [Language Navigation] No label for Language Navigation 

### ISSUE THREAD

**Issue by LevelAccessA11y** [adobe/aem-core-wcm-components#1671]
**Steps to Reproduce**
1. With NVDA running, navigate to a Language Navigation component, and determine whether it can be distinguished from other navigations on the page.

**Current Behavior**
The Language Navigation does not have an accessible name. When the Language Navigation component is used on a page with other navigation elements, screen reader users cannot easily distinguish its purpose.

The Edit dialog for the Language Navigation component does not provide a field for an accessibility label (as the Navigation component does). 

**Expected behavior/code**
When more than one navigation landmark is used on a page, each should have a unique label. Ensure a mechanism is provided for authors to label the Language Navigation component.

**Environment**
Chrome 91, Windows 10, NVDA 2020.4 
AEM Core Components v. 2.17  

**Affected Users**
Screen reader users

**WCAG SC**
1.3.1 Info and Relationships
2.4.1 Bypass Blocks

**Possible Solution**
Use `aria-labelledby` (if appropriate visible text is present) or `aria-label` attribute to provide a label for the navigation.

**Code snippet**
```
 <nav data-cmp-data-layer="{&quot;languagenavigation-e411fdc67f&quot;:{&quot;@type&quot;:&quot;core-components-examples/components/languagenavigation&quot;,&quot;repo:modifyDate&quot;:&quot;2019-06-06T07:40:17Z&quot;}}" id="languagenavigation-e411fdc67f" class="cmp-languagenavigation">
```

**Additional context**
ARIA11: Using ARIA landmarks to identify regions of a page -- https://www.w3.org/TR/2016/NOTE-WCAG20-TECHS-20161007/ARIA11

**Comment by bpauli** [comment]
@adobe export issue to Jira project SITES

**Comment by bpauli** [comment]
done in https://github.com/adobe/aem-core-wcm-components/pull/1701
